docs/preview/features/databricks/databricks-info-provider.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Interact with Databricks to gain insights"
+layout: default
+---
+
+# Interact with Databricks to gain insights
+
+## Installation
+
+To use these features, you have to install the following package:
+
+	```shell
+	PM > Install-Package Arcus.BackgroundJobs.Databricks
+	```
+
+## Usage
+The background job makes use of the available `DatabricksInfoProvider` to measure and report the finished job outcomes.
+This provider can be used outside the background job environment (ex. Azure Functions).
```

Comment:
What about this?
```suggestion
It can be easily setup and used anywhere such as .NET Core workers, Azure Functions and more. We are using this ourselves for our [job metrics](./job-metrics).
```

Reply:
To verify if the link works 😅

Reply:
We'll have to use a hight URL path for this to work. I'll push the change myself.

Reply:
Ah, nevermind, It's good as it is 😅 .